openff/toolkit/utils/ambertools_wrapper.py

```diff
@@ -68,8 +68,7 @@ def is_available():
 
         """
         # TODO: Check all tools needed
-        # TODO: How should we implement find_executable?
```

Comment:
This might be the first TODO that I've ever seen properly resolved :-P

Reply:
I was a bit confused to find it since it was already plenty well implemented 🤷‍♂️  ....